.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "public/assets/osf-assets"]
+	path = public/assets/osf-assets
+	url = https://github.com/brianjgeiger/osf-assets.git
```

Comment:
Perhaps I'll get this transferred over to COS this morning, though I presume it will redirect in any case.